### PR TITLE
[zh-cn]: update the translation of Atomics.add() and sub()

### PR DESCRIPTION
**Pull request by JasonLamv-t**
### Description

Same as the title.

### Motivation

Ditto.

### Additional details

None.

### Related issues and pull requests

#17175

### CODE REVIEW

File: files/zh-cn/web/javascript/reference/global_objects/atomics/add/index.md [hunk 1]
```diff
@@ -1,46 +1,51 @@
...
-```plain
+```js-nolint
 Atomics.add(typedArray, index, value)
 ```
 
 ### 参数
 
 - `typedArray`
-  - : 一个共享的整型 typed array。例如 {{jsxref("Int8Array")}}，{{jsxref("Uint8Array")}}，{{jsxref("Int16Array")}}，{{jsxref("Uint16Array")}}，{{jsxref("Int32Array")}}，或者 {{jsxref("Uint32Array")}}。
+  - : 一个整数类型数组。{{jsxref("Int8Array")}}、{{jsxref("Uint8Array")}}、{{jsxref("Int16Array")}}、{{jsxref("Uint16Array")}}、{{jsxref("Int32Array")}} 或者 {{jsxref("Uint32Array")}} 之一。
```
Comment (jasonren0403) [inline files/zh-cn/web/javascript/reference/global_objects/atomics/add/index.md:23]: 还有 BigInt64Array 和 BigUint64Array

File: files/zh-cn/web/javascript/reference/global_objects/atomics/sub/index.md [hunk 2]
```diff
@@ -1,22 +1,28 @@
...
-```plain
+```js-nolint
 Atomics.sub(typedArray, index, value)
 ```
 
 ### 参数
 
 - `typedArray`
-  - : 一个共享的整型 typed array。例如 {{jsxref("Int8Array")}}, {{jsxref("Uint8Array")}}, {{jsxref("Int16Array")}}, {{jsxref("Uint16Array")}}, {{jsxref("Int32Array")}}, 或者 {{jsxref("Uint32Array")}}.
+  - : 一个整数类型数组。{{jsxref("Int8Array")}}、{{jsxref("Uint8Array")}}、{{jsxref("Int16Array")}}、{{jsxref("Uint16Array")}}、{{jsxref("Int32Array")}} 或者 {{jsxref("Uint32Array")}} 之一。
```
Comment (jasonren0403) [inline files/zh-cn/web/javascript/reference/global_objects/atomics/sub/index.md:23]: 同上

File: files/zh-cn/web/javascript/reference/global_objects/atomics/add/index.md [hunk 3]
```diff
@@ -1,46 +1,51 @@
...
 - `index`
-  - : `typedArray` 中的位置，该位置数值会被加总并更新。
+  - : `typedArray` 中的要加上 `value` 的位置。
 - `value`
   - : 增加的数字。
```
Comment (jasonren0403) [inline files/zh-cn/web/javascript/reference/global_objects/atomics/add/index.md:27]: 对齐 sub

```suggestion
  - : 要增加的数字。
```

**Review by jasonren0403 — COMMENTED** [review]
一些建议

File: files/zh-cn/web/javascript/reference/global_objects/atomics/add/index.md (hunk 1 above)
Comment (JasonLamv-t) [inline files/zh-cn/web/javascript/reference/global_objects/atomics/add/index.md:23]:
```suggestion
  - : 一个整数类型数组。{{jsxref("Int8Array")}}、{{jsxref("Uint8Array")}}、{{jsxref("Int16Array")}}、{{jsxref("Uint16Array")}}、{{jsxref("Int32Array")}}、{{jsxref("Uint32Array")}}、{{jsxref("BigInt64Array")}} 或 {{jsxref("BigUint64Array")}} 之一。
```

File: files/zh-cn/web/javascript/reference/global_objects/atomics/sub/index.md (hunk 2 above)
Comment (JasonLamv-t) [inline files/zh-cn/web/javascript/reference/global_objects/atomics/sub/index.md:23]:
```suggestion
  - : 一个整数类型数组。{{jsxref("Int8Array")}}、{{jsxref("Uint8Array")}}、{{jsxref("Int16Array")}}、{{jsxref("Uint16Array")}}、{{jsxref("Int32Array")}}、{{jsxref("Uint32Array")}}、{{jsxref("BigInt64Array")}} 或 {{jsxref("BigUint64Array")}} 之一。
```

**Review by jasonren0403 — APPROVED** [review]
LGTM 👍

**Pull request merged**